target/linux/ramips/mt7621/base-files/etc/board.d/02_network

```diff
@@ -73,6 +73,9 @@ ramips_setup_interfaces()
 	zyxel,nr7101)
 		ucidef_set_interfaces_lan_wan "lan" "wan"
 		;;
+	humax,e10)
```

Comment:
I think "humax,e10" case can be merged with "winstars,ws-wn583a6" case?

Reply:
Merged!